lib/project_types/theme/messages/messages.rb

```diff
@@ -57,6 +57,7 @@ module Messages
               Usage: {{command:%s theme push [ ROOT ]}}
 
               Options:
+                {{command:-t, --theme=THEME}}     Theme ID or name of the remote theme.
```

Comment:
```suggestion
                {{command:-t, --theme=NAME_OR_ID}}     Theme ID or name of the remote theme.
```

Seems clearer to me.

Reply:
Great suggestion! Fixed :)